crates/admin/protobuf/cluster_ctrl_svc.proto

```diff
@@ -89,7 +89,8 @@ message CreatePartitionSnapshotRequest { uint32 partition_id = 1; }
 
 message CreatePartitionSnapshotResponse {
   string snapshot_id = 1;
-  uint64 min_applied_lsn = 2;
+  uint32 log_id = 2;
+  uint64 min_applied_lsn = 3;
```

Comment:
I only just introduced this in https://github.com/restatedev/restate/pull/2743 so feel okay fudging this - purists may disagree 😇

Reply:
If it wasn't part of any release then I think it's okay